docs/releases/v2018.2.4.rst

```diff
@@ -0,0 +1,52 @@
+Gluon 2018.2.4
+==============
+
+Important Note
+~~~~~~~~~~~~~~
+
+This release will be the last one of the Gluon branch v2018.2.x , an update to v2019.1.x is strongly recommended and should be fairly easy.
```

Comment:
```suggestion
This will be the final release of the v2018.2.x series. Updating to the v2019.1.x release series is the recommended course of action, which should be fairly easy.
```